docs/src/content/docs/es/components/badges.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Insignias
+description: Aprende a usar insignias en Starlight para mostrar información adicional.
+---
+
+import { Badge } from '@astrojs/starlight/components';
+
+Para mostrar pequeñas piezas de información, como un estado o categoría, usa el componente `<Badge>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Badge slot="preview" text="Nuevo" />
+
+</Preview>
+
+## Importación
+
+```tsx
+import { Badge } from '@astrojs/starlight/components';
+```
+
+## Uso
+
+Muestra una insignia usando el componente `<Badge>` y pasa el contenido que deseas mostrar al atributo [`text`](#text) del componente `<Badge>`.
+
+Por defecto, la insignia usará el color de acento del tema de tu sitio.
+Para usar un color de insignia integrado, establece el atributo [`variant`](#variant) en uno de los valores admitidos.
+
+<Preview>
+
+```mdx
+import { Badge } from '@astrojs/starlight/components';
+
+<Badge text="Nota" variant="note" />
+<Badge text="Éxito" variant="success" />
+<Badge text="Tip" variant="tip" />
```

Comment:
```suggestion
<Badge text="Consejo" variant="tip" />
```

Was this missed or expected? Took the translation from https://github.com/withastro/starlight/blob/main/packages/starlight/translations/es.json#L24C17-L24C24